### ISSUE TITLE
Remove Box Colliders in Ui and add Highlight Color

### ISSUE THREAD

**Issue by runette** [ViRGIS-Team/ViRGIS_V1#111]
Box Colliders no longer needed after #110.

Adding a Highlight Color to all UI Elements (i.e. color on mouse over and now on Pointer Over as well) will make the VR UX more intuitive since this is now the only feedback that the UI Element is engaged.
